js/app/FileForm.js

```diff
@@ -123,17 +123,15 @@ $.extend( FileForm.prototype, {
 		self._api.getAsset( prefixedFilename, wikiUrl )
 			.done( function( asset ) {
 				if( asset instanceof WikiAsset ) {
-					if( asset.getLicence() !== null && asset.getLicence().isInGroup( 'unsupported' ) ) {
+					if(	$.inArray( asset.getMediaType(), config.supportedMediaTypes ) === -1 ) {
```

Comment:
Looks good and everything, but I am curious why so many whitespaces after the parenthesis? Just for alignment?